Orange/statistics/util.py

```diff
@@ -231,3 +231,70 @@ def weighted_mean():
             np.zeros(X.shape[1]),
             nans,
             X.shape[0] - nans))
+
+
+def _sparse_has_zeros(x):
+    """ Check if sparse matrix contains any implicit zeros. """
+    return np.prod(x.shape) != x.data.size
```

Comment:
Corrected. Though, methods still won't work for for `dox_matrix` since we rely on `x.data` elsewhere.